deen.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "GiovanYCringe",
+        "oem": "Motorola",
+        "device": "Motorola one",
```

Comment:
Try "one" with capital letter "O" not "o"

Reply:
Done